graphdatascience/graph/graph_proc_runner.py

```diff
@@ -167,6 +168,11 @@ def graphProperty(self) -> GraphPropertyRunner:
         self._namespace += ".graphProperty"
         return GraphPropertyRunner(self._query_runner, self._namespace, self._server_version)
 
+    @property
+    def nodeLabel(self) -> GraphLabelRunner:
+        self._namespace = "gds.alpha.graph.nodeLabel"
```

Comment:
```suggestion
       self._namespace += ".nodeLabel"
```
This makes sure that users cannot leave out `alpha` for example from their method call